nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -360,6 +360,19 @@
           <literal>rxvt-unicode</literal> explicitly.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The linux kernel package infrastructure was moved out of
+          <literal>all-packages.nix</literal>, and restructured. Linux
+          related functions and attributes now live under the
+          <literal>pkgs.linuxKernel</literal> attribute set. The
+          versioned <literal>linuxPackages_*</literal> package sets and
+          kernels from <literal>pkgs</literal> are now considered
+          deprecated, and kept for backwards compatibility only. If you
```

Comment:
This contradicts this line elsewhere:

> While `pkgs.linuxKernel.packages` contains all available kernel packages, you may want to use one of the `pkgs.linuxPackages_*` aliases, that are kept up to date with new versions.

It can't be deprecated and recommended at the same time.

Reply:
The intention is to deprecate versioned aliases and keep / recommend use of unversioned aliases.